tla-pp/src/main/scala/at/forsyte/apalache/tla/pp/Unroller.scala

```diff
@@ -4,38 +4,84 @@ import at.forsyte.apalache.tla.lir.aux.{ExceptionOrValue, FailWith, SucceedWith}
 import at.forsyte.apalache.tla.lir._
 import at.forsyte.apalache.tla.lir.oper.TlaOper
 import at.forsyte.apalache.tla.lir.storage.{BodyMap, BodyMapFactory}
-import at.forsyte.apalache.tla.lir.transformations.standard.InlinerOfUserOper
+import at.forsyte.apalache.tla.lir.transformations.standard.{IncrementalRenaming, InlinerOfUserOper, ReplaceFixed}
 import at.forsyte.apalache.tla.lir.transformations.{TlaExTransformation, TlaModuleTransformation, TransformationTracker}
 import at.forsyte.apalache.tla.lir.values.TlaInt
 
-class Unroller( tracker : TransformationTracker ) extends TlaModuleTransformation {
+/**
+  * Replaces definitions of user-defined recursive operators with a finite-step unrolling.
+  *
+  * For each recursive operator A the user must define two additional TLA+ operators,
+  *  - [UNROLL_TIMES_PREFIX]_A
+  *  - [UNROLL_DEFAULT_PREFIX]_A
+  *
```

Comment:
Does this markdown-like commenting work in javadoc? I think you have to use a bit of HTML here, e.g., `<p>...</p>` and `<ul><li>...</li></ul>`.